provisioning/ubuntu-provision.sh

```diff
@@ -553,14 +553,27 @@ function install_updatecli() {
 function install_awscli() {
```

Comment:
(seen in pair: add a comment explaining the 2 different behavior of the function)